cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -26,19 +26,584 @@ Cask Data Application Platform Release Notes
+  JAVA_HOME is defined as a path with spaces in it. A workaround is to use a definition of 
+  JAVA_HOME that does not include spaces.
+
+- `CDAP-3665 <https://issues.cask.co/browse/CDAP-3665>`__ -
+  When a MapReduce job fails, CDAP says all mappers/reducers were successful.
+  
+- `CDAP-3689 <https://issues.cask.co/browse/CDAP-3689>`__ -
+  Metadata entity types should be lower-case, not upper-case.
+  
+- `CDAP-3697 <https://issues.cask.co/browse/CDAP-3697>`__ -
+  CDAP Explore is broken on secure CDH 5.1.
+  
+- `CDAP-3698 <https://issues.cask.co/browse/CDAP-3698>`__ -
+  CDAP Explore is unable to get a delegation token while fetching next results on HDP2.0.
+  
+- `CDAP-3720 <https://issues.cask.co/browse/CDAP-3720>`__ -
+  The WorklowAction configure method is invoked multiple times.
+  
 - `CDAP-3732 <https://issues.cask.co/browse/CDAP-3732>`__ -
   The first time a flow is started in a namespace, there will be an error message in the logs
   about a failure to enable explore on the system.queue table. A call to enable exploration
   on the table should not be made, as it is not meant to be explorable.
 
+- `CDAP-3744 <https://issues.cask.co/browse/CDAP-3744>`__ -
+  The HTTP RESTful endpoints to enable explore are not documented.
+  
+- `CDAP-3749 <https://issues.cask.co/browse/CDAP-3749>`__ -
+  The DBSource plugin does not allow a username with an empty password.
+  
+- `CDAP-3750 <https://issues.cask.co/browse/CDAP-3750>`__ -
+  If a table schema contains a field name that is a reserved word in Hive DDL, 'enable explore' fails.
+  
+- `CDAP-3752 <https://issues.cask.co/browse/CDAP-3752>`__ -
+  The confirmation dialogs across the CDAP-UI are not standardized.
+  
+- `CDAP-3756 <https://issues.cask.co/browse/CDAP-3756>`__ -
+  Refreshing the CDAP UI from any page after it has been idle for a while takes you back to the home page
+  
+- `CDAP-3757 <https://issues.cask.co/browse/CDAP-3757>`__ -
+  Order-by using an aliased column fails in Explore on Standalone.
+  
+- `CDAP-3810 <https://issues.cask.co/browse/CDAP-3810>`__ -
+  org.apache.activemq.ActiveMQConnectionFactory cannot be cast to javax.jms.ConnectionFactory; a java.lang.ClassCastException is thrown.
+  
+- `CDAP-3815 <https://issues.cask.co/browse/CDAP-3815>`__ -
+  The S3BatchSource plugin's file regex doesn't filter the files to be read by the source.
+  
 - `CDAP-3817 <https://issues.cask.co/browse/CDAP-3817>`__ -
-  On initial startup, after a fresh CDAP installation, it is possible for the ``data.tx.snapshot.dir``
-  HDFS directory to be created with improper permissions.
+  Race condition in the creation of data.tx.snapshot.dir can occur. On initial startup,
+  after a fresh CDAP installation, it is possible for the ``data.tx.snapshot.dir`` HDFS
+  directory to be created with improper permissions.
+  
+- `CDAP-3818 <https://issues.cask.co/browse/CDAP-3818>`__ -
+  The TimeFilter documentation is inconsistent.
```

Comment:
Well, to be honest, like many software companies, we have been parsimonious in our "known issues".